src/app/hooks/use-notifications.ts

```diff
@@ -122,6 +123,12 @@ const deleteNotificationsByVersion = (env: Environment) => ({ version }: { versi
 		});
 };
 
+const sortTransactionNotificationsDesc = (notifications: any[]) =>
```

Comment:
you can probably use https://github.com/ArkEcosystem/utils/blob/master/src/sort-by-desc.ts for simplicity here instead

```
import { sortByDesc } from "@arkecosystem/utils";

sortByDesc(notifications, (notification) => notification.meta.timestamp)
```

Reply:
@dated you are absolutely right. Thanks :+1: